testing/infrastructure/InfrastructureComponentShape.ttl

```diff
@@ -27,34 +27,35 @@ shapes:InfrastructureComponentShape
   sh:property [
   	a sh:PropertyShape ;
   	sh:path ids:maintainer ;
-		#sh:or (
-	  #  [ sh:class ids:Participant ; ]
-	  #  [
-		sh:nodeKind sh:IRI ;# ]
-		#) ;
+		sh:or (
+	    [ sh:class ids:Participant ; sh:order 2 ; ]
+	    [	sh:nodeKind sh:IRI ; sh:order 1 ; ]
+		) ;
     sh:minCount 1 ;
+  	sh:maxCount 1 ;
   	sh:severity sh:Violation ;
   	sh:message "<https://raw.githubusercontent.com/International-Data-Spaces-Association/InformationModel/master/testing/infrastructure/InfrastructureComponentShape.ttl> (InfrastructureComponentShape): An ids:maintainer property must have at least one point from an ids:InfrastructureComponent to an ids:Participant."@en ;
   ] ;
 
   sh:property [
     a sh:PropertyShape ;
     sh:path ids:curator ;
-		#sh:or (
-	  #  [ sh:class ids:Participant ; ]
-	  #  [
-		sh:nodeKind sh:IRI ; #]
-		#) ;
-    sh:minCount 1 ;
+		sh:or (
+	    [ sh:class ids:Participant ; sh:order 2 ; ]
+	    [ sh:nodeKind sh:IRI ; sh:order 1 ; ]
+		) ;
+    #sh:minCount 1 ;
+  	sh:maxCount 1 ;
     sh:severity sh:Violation ;
     sh:message "<https://raw.githubusercontent.com/International-Data-Spaces-Association/InformationModel/master/testing/infrastructure/InfrastructureComponentShape.ttl> (InfrastructureComponentShape): An ids:curator property must have at least one point from an ids:InfrastructureComponent to an ids:Participant."@en ;
   ] ;
 
   sh:property [
-   a sh:PropertyShape ;
-   sh:path ids:physicalLocation ;
-   sh:class ids:Location ;
-   sh:severity sh:Violation ;
+		a sh:PropertyShape ;
+		sh:path ids:physicalLocation ;
+		sh:class ids:Location ;
+		sh:maxCount 1 ;
```

Comment:
Why was there a `sh:maxCount 1` introduced to this file but not to the `CatalogueShape.ttl`? Looks inconsistent at a first glance.

Reply:
Don't remember the details anymore... But having more than one physicalLocation seems unrealisitc, doesn't it?

Reply:
Good point, even I do not remember the details 😁